.github/workflows/pull-request-check.yml

```diff
@@ -166,3 +166,89 @@ jobs:
         with:
           name: che-dev
           path: che-dev.tgz
+
+  publish-images:
+    name: publish image from the pull request
+    needs: [assemble, dev]
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout che-code source code
+        uses: actions/checkout@v4
+
+      - name: Download Pull Request Number artifact
+        uses: actions/download-artifact@v4
```

Comment:
the job failed on this step, please see https://github.com/che-incubator/che-code/actions/runs/13438479460/job/37552694049#step:3:9